tests/test_contracts.py

```diff
@@ -14,3 +15,21 @@ def test_contract_manager():
     assert manager.get_event_abi('TokenNetwork', 'ChannelOpened')
     with pytest.raises(KeyError):
         manager.get_event_abi('TokenNetwork', 'NonExistant')
+
+
+def test_deploy_multiple_tokens(add_and_register_token):
+    """Deploy and register $DEPLOY_TOKENS tokens in a TokenNetworksRegistry"""
+    DEPLOY_TOKENS = 10
```

Comment:
think it would be better to use parametrization here: https://docs.pytest.org/en/latest/parametrize.html

Reply:
idk, I don't see how this test could benefit from using multiple parameters, it's quite deterministic. Only if you go above certain number of deployed networks eth tester will start having hard time returning log events.

Reply:
I'm fine with either, just thought it would be nicer than defining the number in the test. Do as you like.